tests/FunctionsTest.php

```diff
@@ -34,44 +36,52 @@ public function testCopiesToStream()
         $s1 = Psr7\stream_for('foobaz');
         $s2 = Psr7\stream_for('');
         Psr7\copy_to_stream($s1, $s2);
-        $this->assertEquals('foobaz', (string) $s2);
+        $this->assertEquals('foobaz', (string)$s2);
```

Comment:
Why was the string cast changed in some places? It's not consistent with the rest and makes merging branches a pain...